spec/ibm/cloud/sdk/vpc_spec.rb

```diff
@@ -24,9 +24,9 @@
   vpn_gateways: IBM::Cloud::SDK::VPC::VPNGateways
 }.freeze
 
-vpc = IBM::CloudSDK.new(ENV['IBM_CLOUD_APIKEY']).vpc
```

Comment:
Moved this into an rspec context so that just loading the rspec file didn't raise an exception